google-cloud-tools-plugin/testSrc/com/google/cloud/tools/intellij/appengine/cloud/flexible/AppEngineFlexibleDeploymentEditorTest.java

```diff
@@ -400,6 +400,7 @@ public void testDockerfileStartsDisabledAndWithModuleSetting() {
 
   @Override
   public void tearDown() throws Exception {
+    editor.getModulesWithFlexFacetComboBox().removeAllItems();
```

Comment:
I'm worried that the editor isn't getting disposed because of this. 

I analyzed the heap dump and found paths to gc roots from this combobox. Its type is `JComboBox<Module>` which makes me suspect that it retains references to the modules causing the corresponding project to leak.